src/Network/Network.Test/ScenarioTests/CortexTests.ps1

```diff
@@ -481,4 +481,208 @@ function Test-CortexExpressRouteCRUD
     {
         Clean-ResourceGroup $rgname
     }
+}
+
+<# .SYNOPSIS
+ Point to site Cortex feature tests
+ #>
+ function Test-P2SCortexCRUD
+ {
+ param 
+    ( 
+        $basedir = ".\" 
+    )
+
+    # Setup
+    $rgname = Get-ResourceGroupName
+    $rglocation = Get-ProviderLocation "Microsoft.Network/VirtualWans"
+ 
+    $virtualWanName = Get-ResourceName
+    $virtualHubName = Get-ResourceName
+    $VpnServerConfiguration1Name = Get-ResourceName
+    $VpnServerConfiguration2Name = Get-ResourceName
+    $P2SVpnGatewayName = Get-ResourceName
+    $vpnclientAuthMethod = "EAPTLS"
+ 	
+	$storeName = 'blob' + $rgName
```

Comment:
I'll be best if you could format your changes somehow... like indent correctly

Reply:
I think I did.